android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -183,15 +185,47 @@ public void onHostResume() {
         /*
          * In case it wasn't set.
          */
-        if (themedReactContext.getCurrentActivity() != null) {
-            themedReactContext.getCurrentActivity()
-                              .setVolumeControlStream(AudioManager.STREAM_VOICE_CALL);
+        if (themedReactContext.getCurrentActivity() != null && cameraCapturer != null) {
+            /*
+            * If the local video track was released when the app was put in the background, recreate.
+            */
+            if (localVideoTrack == null) {
+                localVideoTrack = LocalVideoTrack.create(getContext(), true, cameraCapturer);
+                if (thumbnailVideoView != null && localVideoTrack != null) {
+                    localVideoTrack.addRenderer(thumbnailVideoView);
+                }
+
+                /*
+                * If connected to a Room then share the local video track.
+                */
+                if (localParticipant != null) {
+                    localParticipant.addVideoTrack(localVideoTrack);
+                }
+                themedReactContext.getCurrentActivity().setVolumeControlStream(AudioManager.STREAM_VOICE_CALL);
```

Comment:
You've changed this to only get called when cameraCapturer is not null, but maybe they don't have a camera active but *do* have audio capturing active in which case you'd still want to run this right?